torch/testing/_internal/common_nn.py

```diff
@@ -2531,7 +2531,7 @@ def unsqueeze_inp(inp):
         check_gradgrad=False,
         desc='gelu_activation',
         with_tf32=True,
-        tf32_precision=0.05,
```

Comment:
Agree with @janeyx99's comment, it seems like this will change the precision for these tests on any environment. Is it possible to make this more specific to H100

Reply:
Just added a SM90OrLater check, so we can have the updated precision for H100 and the old precision for everything else